frontend/android/src/main/java/io/github/droidkaigi/confsched2019/ui/MainActivity.kt

```diff
@@ -85,6 +87,16 @@ class MainActivity : DaggerAppCompatActivity() {
                 userActionCreator.load()
             }
         }
+        systemStore.systemLanguageChanged.changed(this) {
+            Timber.debug { "detect language changed to $it" }
+            forceReloadCurrentDestination()
+        }
+    }
+
+    private fun forceReloadCurrentDestination() {
+        if (!navController.popBackStack(R.id.main, true)) {
```

Comment:
I don't have confidence as to whether this is OK.
Please let me know if there is a better way 🙏